app/services/merge_patient_service.rb

```diff
@@ -0,0 +1,33 @@
+class MergePatientService
```

Comment:
I'm wondering if this shouldn't just live in the `Patient` model...

Reply:
Unsure about this right now. Since there's a lot of room for reuse of this kind of code across models, perhaps we can change this design after we implement sync for BP, and Users as well.